docs/en/sql-reference/functions/string-search-functions.md

```diff
@@ -536,4 +536,44 @@ For case-insensitive search or/and in UTF-8 format use functions `ngramSearchCas
 !!! note "Note"
     For UTF-8 case we use 3-gram distance. All these are not perfectly fair n-gram distances. We use 2-byte hashes to hash n-grams and then calculate the (non-)symmetric difference between these hash tables – collisions may occur. With UTF-8 case-insensitive format we do not use fair `tolower` function – we zero the 5-th bit (starting from zero) of each codepoint byte and first bit of zeroth byte if bytes more than one – this works for Latin and mostly for all Cyrillic letters.
 
+## substringCount(haystack, needle) {#substringCount}
+
+Count the number of substring occurrences
+
+For a case-insensitive search, use the function `substringCountCaseInsensitive`.
```

Comment:
Is there `CaseInsensitiveUTF8` variant?

Reply:
In the latest version - yes (3265b29c19c2413f5275d8b05271299e718beea0)